makefile.checks

```diff
@@ -80,7 +100,16 @@ $(error LaL is set and requires XMLADA to be installed. \
 endif
 endif
 
-ifneq ($(SOCKET), std)
+ifeq ($(SOCKET), std)
+ifeq ($(SSL_DYNAMIC), true)
+# SSL_DYNAMIC can be true only together with SOCKET in openssl | gnutls
+SSL_DYNAMIC = false
```

Comment:
looks like some debug code here, no?

Reply:
No. It mean that if SSL_DYNAMIC defined in makefile.setup as true and not defined in command line then we turn it into false. Next line is comparison SSL_DYNAMIC with true. The SSL_DYNAMIC can be true after SSL_DYNAMIC = false only if the SSL_DYNAMIC=true defined in command line.